ruby_connectors/lib/ruby_connectors/duckdb_connector/client.rb

```
@@ -6,21 +6,23 @@ class Client < Core::BaseConnector
       def connect
         @connection.connect
       end
 
-      # def read(stream_name, page = 1, per_page = 30)
-      #   @reader.read(stream_name, page, per_page)
-      # end
-
-      # def write(data)
-      #   @writer.write(data)
-      # end
+      def write(data, table_name:, schema:, schema_name: nil, database_name: nil, primary_keys: nil)
```

rubocop (1.69.1)

[convention] Avoid parameter lists longer than 5 parameters. [6/5]

(Metrics/ParameterLists)